src/utils/loaders/getCityFeed.test.js

```diff
@@ -1,4 +1,27 @@
-const problems = require('./getCityFeed')
-test('first test to Add function', () => {
-    expect(problems.add(1,2)).toEqual(3)
-})
\ No newline at end of file
+import { getCityFeed } from './getCityFeed'
+
+describe('getCityFeed function', () => {
+  // Mocking fetch
+  global.fetch = jest.fn(() =>
+    Promise.resolve({
+      json: () => Promise.resolve({ mockData: 'mockValue' }),
+    })
+  )
+
+  beforeEach(() => {
+    fetch.mockClear()
+  })
+
+  it('should call fetch with the correct URL', async () => {
+    const token = 'mockToken'
+    const expectedURL = `https://api.waqi.info/feed/here/?token=${token}`
```

Comment:
url from ENV file, please. 

Please note: we are gonna use axios, we are gonna replace for axios next sprint

Reply:
updated code and test to use axios

the whole URL is now from ENV file, not jus the token part (that was already from ENV file)